requirements/reproducible-linux.txt

```diff
@@ -930,6 +930,10 @@ types-awscrt==0.19.1 \
     --hash=sha256:61833aa140e724a9098025610f4b8cde3dcf65b842631d7447378f9f5db4e1fd \
     --hash=sha256:68fffeb75396e9e7614cd930b2d52295f680230774750907bcafb56f11514043
     # via botocore-stubs
+types-python-dateutil==2.8.19.14 \
```

Comment:
Pyinstaller checks were failing because `types-python-dateutil==2.8.19.14` was not added by dependabot when updating `arrow`. The reproducibles GHA has been updated to run when these reproducible files change as well.